policygenerator/policy-sets/community/openshift-plus-setup/machine-sets.yaml

```
@@ -77,6 +77,17 @@ spec:
                     - name: tag:Name
                       values:
                       - '{{ (lookup "config.openshift.io/v1" "Infrastructure" "" "cluster").status.infrastructureName }}-worker-sg'
+                  {{- $ocpversion := semver (fromClusterClaim "version.openshift.io") }}
+                  {{- if gt $ocpversion.Minor 15 }}
```

We have `semverCompare` available--maybe use it directly?
```suggestion
                  {{- semverCompare ">4.15" (fromClusterClaim "version.openshift.io") }}
```

---

I saw the semverCompare function, but I wasn't sure how it handled the patch release if you did a compare as described.  I didn't play with it -- just went down the path that made sense at the moment.  I'll test this some and see what happens.

---

Oh, you're probably right. It probably should be `">=4.16"`

---

When I use any of the comparisons with 4.16.0-rc.2 builds the comparisons just don't seem to work.  While it works fine if I change the comparison to run against just "4.16.0", I think I am leaning towards keeping the existing code rather than switch to semverCompare which appears to have trouble with some OCP versioning - even if these are rc builds.

---

Works for me! And I guess the risk of bumping to OCP v5 is fairly low right now 🙂